// blockchain-addressbook/build/address-book/base/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.base = void 0;
var beefyfinance_1 = require("./platforms/beefyfinance");
var balancer_1 = require("./platforms/balancer");
var baseSwap_1 = require("./platforms/baseSwap");
var bvm_1 = require("./platforms/bvm");
var aerodrome_1 = require("./platforms/aerodrome");
var equalizer_1 = require("./platforms/equalizer");
var tokens_1 = require("./tokens/tokens");
var convertSymbolTokenMapToAddressTokenMap_1 = require("../../util/convertSymbolTokenMapToAddressTokenMap");
var _base = {
    platforms: {
        beefyfinance: beefyfinance_1.beefyfinance,
        balancer: balancer_1.balancer,
        baseSwap: baseSwap_1.baseSwap,
        bvm: bvm_1.bvm,
        aerodrome: aerodrome_1.aerodrome,
        equalizer: equalizer_1.equalizer,
    },
    tokens: tokens_1.tokens,
    tokenAddressMap: (0, convertSymbolTokenMapToAddressTokenMap_1.convertSymbolTokenMapToAddressTokenMap)(tokens_1.tokens),
};
exports.base = _base;
