// blockchain-addressbook/build/address-book/arbitrum/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.arbitrum = void 0;
var beefyfinance_1 = require("./platforms/beefyfinance");
var sushi_1 = require("./platforms/sushi");
var swapfish_1 = require("./platforms/swapfish");
var balancer_1 = require("./platforms/balancer");
var solidlizard_1 = require("./platforms/solidlizard");
var ramses_1 = require("./platforms/ramses");
var arbidex_1 = require("./platforms/arbidex");
var chronos_1 = require("./platforms/chronos");
var bunni_1 = require("./platforms/bunni");
var tokens_1 = require("./tokens/tokens");
var convertSymbolTokenMapToAddressTokenMap_1 = require("../../util/convertSymbolTokenMapToAddressTokenMap");
var _arbitrum = {
    platforms: {
        beefyfinance: beefyfinance_1.beefyfinance,
        sushi: sushi_1.sushi,
        swapfish: swapfish_1.swapfish,
        balancer: balancer_1.balancer,
        solidlizard: solidlizard_1.solidlizard,
        ramses: ramses_1.ramses,
        arbidex: arbidex_1.arbidex,
        chronos: chronos_1.chronos,
        bunni: bunni_1.bunni,
    },
    tokens: tokens_1.tokens,
    tokenAddressMap: (0, convertSymbolTokenMapToAddressTokenMap_1.convertSymbolTokenMapToAddressTokenMap)(tokens_1.tokens),
};
exports.arbitrum = _arbitrum;
