// blockchain-addressbook/build/address-book/index.js
"use strict";
var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    var desc = Object.getOwnPropertyDescriptor(m, k);
    if (!desc || ("get" in desc ? !m.__esModule : desc.writable || desc.configurable)) {
      desc = { enumerable: true, get: function() { return m[k]; } };
    }
    Object.defineProperty(o, k2, desc);
}) : (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    o[k2] = m[k];
}));
var __exportStar = (this && this.__exportStar) || function(m, exports) {
    for (var p in m) if (p !== "default" && !Object.prototype.hasOwnProperty.call(exports, p)) __createBinding(exports, m, p);
};
var _a;
Object.defineProperty(exports, "__esModule", { value: true });
exports.addressBookByChainId = exports.addressBook = void 0;
var polygon_1 = require("./polygon");
var bsc_1 = require("./bsc");
var avax_1 = require("./avax");
var fantom_1 = require("./fantom");
var heco_1 = require("./heco");
var one_1 = require("./one");
var arbitrum_1 = require("./arbitrum");
var celo_1 = require("./celo");
var moonriver_1 = require("./moonriver");
var cronos_1 = require("./cronos");
var aurora_1 = require("./aurora");
var fuse_1 = require("./fuse");
var metis_1 = require("./metis");
var moonbeam_1 = require("./moonbeam");
var emerald_1 = require("./emerald");
var optimism_1 = require("./optimism");
var kava_1 = require("./kava");
var ethereum_1 = require("./ethereum");
var canto_1 = require("./canto");
var zksync_1 = require("./zksync");
var zkevm_1 = require("./zkevm");
var base_1 = require("./base");
var gnosis_1 = require("./gnosis");
var linea_1 = require("./linea");
var mantle_1 = require("./mantle");
var chainid_1 = require("../types/chainid");
__exportStar(require("../types/chainid"), exports);
var _addressBook = {
    polygon: polygon_1.polygon,
    bsc: bsc_1.bsc,
    avax: avax_1.avax,
    fantom: fantom_1.fantom,
    heco: heco_1.heco,
    one: one_1.one,
    arbitrum: arbitrum_1.arbitrum,
    celo: celo_1.celo,
    moonriver: moonriver_1.moonriver,
    cronos: cronos_1.cronos,
    aurora: aurora_1.aurora,
    fuse: fuse_1.fuse,
    metis: metis_1.metis,
    moonbeam: moonbeam_1.moonbeam,
    emerald: emerald_1.emerald,
    optimism: optimism_1.optimism,
    kava: kava_1.kava,
    ethereum: ethereum_1.ethereum,
    canto: canto_1.canto,
    zksync: zksync_1.zksync,
    zkevm: zkevm_1.zkevm,
    base: base_1.base,
    gnosis: gnosis_1.gnosis,
    linea: linea_1.linea,
    mantle: mantle_1.mantle,
};
var _addressBookByChainId = (_a = {},
    _a[chainid_1.ChainId.polygon] = polygon_1.polygon,
    _a[chainid_1.ChainId.bsc] = bsc_1.bsc,
    _a[chainid_1.ChainId.avax] = avax_1.avax,
    _a[chainid_1.ChainId.fantom] = fantom_1.fantom,
    _a[chainid_1.ChainId.heco] = heco_1.heco,
    _a[chainid_1.ChainId.one] = one_1.one,
    _a[chainid_1.ChainId.arbitrum] = arbitrum_1.arbitrum,
    _a[chainid_1.ChainId.celo] = celo_1.celo,
    _a[chainid_1.ChainId.moonriver] = moonriver_1.moonriver,
    _a[chainid_1.ChainId.cronos] = cronos_1.cronos,
    _a[chainid_1.ChainId.aurora] = aurora_1.aurora,
    _a[chainid_1.ChainId.fuse] = fuse_1.fuse,
    _a[chainid_1.ChainId.metis] = metis_1.metis,
    _a[chainid_1.ChainId.moonbeam] = moonbeam_1.moonbeam,
    _a[chainid_1.ChainId.emerald] = emerald_1.emerald,
    _a[chainid_1.ChainId.optimism] = optimism_1.optimism,
    _a[chainid_1.ChainId.kava] = kava_1.kava,
    _a[chainid_1.ChainId.ethereum] = ethereum_1.ethereum,
    _a[chainid_1.ChainId.canto] = canto_1.canto,
    _a[chainid_1.ChainId.zksync] = zksync_1.zksync,
    _a[chainid_1.ChainId.zkevm] = zkevm_1.zkevm,
    _a[chainid_1.ChainId.base] = base_1.base,
    _a[chainid_1.ChainId.gnosis] = gnosis_1.gnosis,
    _a[chainid_1.ChainId.linea] = linea_1.linea,
    _a[chainid_1.ChainId.mantle] = mantle_1.mantle,
    _a);
exports.addressBook = _addressBook;
exports.addressBookByChainId = _addressBookByChainId;
