// blockchain-addressbook/build/address-book/ethereum/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.ethereum = void 0;
var beefyfinance_1 = require("./platforms/beefyfinance");
var balancer_1 = require("./platforms/balancer");
var aura_1 = require("./platforms/aura");
var sushi_1 = require("./platforms/sushi");
var synapse_1 = require("./platforms/synapse");
var solidly_1 = require("./platforms/solidly");
var verse_1 = require("./platforms/verse");
var gamma_1 = require("./platforms/gamma");
var tokens_1 = require("./tokens/tokens");
var convertSymbolTokenMapToAddressTokenMap_1 = require("../../util/convertSymbolTokenMapToAddressTokenMap");
var _ethereum = {
    platforms: {
        beefyfinance: beefyfinance_1.beefyfinance,
        balancer: balancer_1.balancer,
        aura: aura_1.aura,
        sushi: sushi_1.sushi,
        synapse: synapse_1.synapse,
        solidly: solidly_1.solidly,
        verse: verse_1.verse,
        gamma: gamma_1.gamma,
    },
    tokens: tokens_1.tokens,
    tokenAddressMap: (0, convertSymbolTokenMapToAddressTokenMap_1.convertSymbolTokenMapToAddressTokenMap)(tokens_1.tokens),
};
exports.ethereum = _ethereum;
