// blockchain-addressbook/build/address-book/fantom/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.fantom = void 0;
var beefyfinance_1 = require("./platforms/beefyfinance");
var beethovenx_1 = require("./platforms/beethovenx");
var spookyswap_1 = require("./platforms/spookyswap");
var spiritswap_1 = require("./platforms/spiritswap");
var sushiFtm_1 = require("./platforms/sushiFtm");
var solidly_1 = require("./platforms/solidly");
var tombswap_1 = require("./platforms/tombswap");
var solace_1 = require("./platforms/solace");
var equalizer_1 = require("./platforms/equalizer");
var fvm_1 = require("./platforms/fvm");
var tokens_1 = require("./tokens/tokens");
var convertSymbolTokenMapToAddressTokenMap_1 = require("../../util/convertSymbolTokenMapToAddressTokenMap");
var _fantom = {
    platforms: {
        beefyfinance: beefyfinance_1.beefyfinance,
        beethovenx: beethovenx_1.beethovenx,
        spookyswap: spookyswap_1.spookyswap,
        spiritswap: spiritswap_1.spiritswap,
        sushiFtm: sushiFtm_1.sushiFtm,
        solidly: solidly_1.solidly,
        tombswap: tombswap_1.tombswap,
        solace: solace_1.solace,
        equalizer: equalizer_1.equalizer,
        fvm: fvm_1.fvm,
    },
    tokens: tokens_1.tokens,
    tokenAddressMap: (0, convertSymbolTokenMapToAddressTokenMap_1.convertSymbolTokenMapToAddressTokenMap)(tokens_1.tokens),
};
exports.fantom = _fantom;
